### DISCUSSION TITLE
Starting a Personal Github Discussion Board | /en/2022/12/github-discussion/

### DISCUSSION THREAD

**Discussion by giscus** [rbind/yihui#1461] [Comments]
Life has been a little difficult for me since the pandemic. The pandemic has had several consequences (e.g., families who need childcare can definitely feel it). Getting COVID was not the worst part. &hellip;

https://yihui.org/en/2022/12/github-discussion/

**Comment by JinjiPang** [comment]
Based on the number of blog posts you wrote, I inferred that you had a tough time in 2012, 2013, 2015 and 2016, especially 2013. I was surprised because I thought finishing your PhD might have made you less stressed. In the last three years, I've seen that 2020 was tough for you, 2021 was okay, and 2022 seems to be the same as before.

**Reply by yihui** [reply]
2012 was an exception. I think I was actually having fun at that time. 2013 was the year when I started working for RStudio and was about to graduate at the same time. The time between 2014 and 2017 was explained in [the career crisis post](https://yihui.org/en/2018/02/career-crisis/), and I didn't write any posts in 2015 or 2016.

I think I'll resume blogging frequently from now on. I've been writing lately because it's holiday time and I don't get as much distraction as I usually do.
